index.html

```diff
@@ -54,5 +54,165 @@ <h6>LET'S CONNECT</h6>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap2.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h1>Multi-Post Stories</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <button><h6>See Project</h6></button>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap3.png" alt="" />
+        </a>
+        <ul class="leftblock">
+          <li .primarytext>
+            <h1>Tonic</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
```

Comment:
- [ ] I would suggest you please provide padding as the javascript buttons are not getting the background. See the image below and fix it.
- [ ] I would suggest you please fix every javascript buttons
### Your design
![hissss](https://user-images.githubusercontent.com/81727988/192098372-001ebe07-151a-4f84-8540-2b950ff5787b.png)